frontend/src/component/admin/users/UsersList/UserSessionsCell/UserSessionsCell.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from 'react';
+import { IconCell } from 'component/common/Table/cells/IconCell/IconCell';
+import WarningIcon from '@mui/icons-material/WarningAmber';
+import { Tooltip } from '@mui/material';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { useVariant } from 'hooks/useVariant';
+
+type UserSessionsCellProps = {
+    count?: number;
+};
+
+export const UserSessionsCell: FC<UserSessionsCellProps> = ({ count }) => {
+    const { uiConfig } = useUiConfig();
+    const minimumCountToShow = useVariant<number>(
+        uiConfig.flags.showUserDeviceCount,
+    );
+
+    if (!count || count < (minimumCountToShow ? minimumCountToShow : 5)) {
+        return null;
+    }
+
+    return (
+        <IconCell
+            icon={
+                <>
+                    <Tooltip title={`Multiple parallel sessions (${count})`}>
```

Comment:
maybe multiple browser sessions? for users it's probably more important that it's many browsers using the same app. It's more important for us that they are parallel.